dev-qt/qt6-location/qt6_location-6.6.0.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="A family of player software for 2d vector graphics animations"
```

Comment:
This doesn't look quite right.

Reply:
Hmm, not sure where I got that :) will  have to run some searches for it (they don't always supply a README with sufficiant information)